src/common/browser-adapters/browser-adapter-event-manager.ts

```diff
@@ -76,19 +95,27 @@ export class BrowserAdapterEventManager {
             result = listener(...eventArgs);
         } catch (error) {
             console.error('Error thrown in application listener: ', error);
```

Comment:
Can this inject a `Logger` dependency and use `logger.error`, rather than calling `console.error` directly? Even if the message isn't important to test, doing that and using a noop logger avoids unnecessary console spew during test execution.

Reply:
good call!